pkg/scheduler/framework/plugins/podtopologyspread/plugin.go

```diff
@@ -35,14 +36,27 @@ const (
 	ErrReasonNodeLabelNotMatch = ErrReasonConstraintsNotMatch + " (missing required label)"
 )
 
+var systemDefaultConstraints = []v1.TopologySpreadConstraint{
```

Comment:
why do we define this here instead of defaults.go?

Reply:
See https://github.com/kubernetes/kubernetes/pull/95048#discussion_r500580974